app/client/src/features/events/DeleteEvent.tsx

```diff
@@ -0,0 +1,156 @@
+        if (!eventId) return; //eventId could be empty
+        commit({
+            variables: { input: { eventId, ...submittedForm } },
+            onCompleted({ deleteEvent }) {
+                if (deleteEvent.isError) {
+                    displaySnack(deleteEvent.message);
+                } else {
+                    displaySnack('Event deleted successfully!');
+                    //forces the page to reload which lets it route back to the proper page
+                    window.location.reload()
+                    //route to list of organizations after successfully deleting event
+                    //TODO: https://github.com/c2c-project/prytaneum/issues/265#issue-1152493329
+                }
+            },
+        });
+    }
+
+    const { handleSubmit, handleChange, values, errors, resetForm } = useFormik<TDeleteEvent>({
+        initialValues: makeInitialState(intialDeleteEvent),
+        validationSchema: deleteEventValidationSchema,
+        // eslint-disable-next-line @typescript-eslint/no-use-before-define
+        onSubmit: handleCommit,
+    });
+
+    return (
+        <Grid container spacing={2}>
+            <Grid item xs={12}>
+                <Typography variant='body2'>All of your event information will be erased from Prytaneum.</Typography>
+            </Grid>
+            <Grid item xs={12}>
+                <Typography variant='body2'>
+                    <b>This action is irreversible.</b> Please enter <b>{eventTitle}</b> below twice to confirm.
+                </Typography>
+            </Grid>
+            <Form className={classes.form} onSubmit={handleSubmit}>
+                <FormContent>
+                    <TextField
+                        inputProps={{ 'aria-label': `Enter ${eventTitle} to delete event` }}
+                        label='Enter event title'
+                        helperText={errors.title}
+                        error={Boolean(errors.title)}
+                        required
+                        variant='outlined'
+                        value={values.title}
+                        onChange={handleChange('title')}
+                        spellCheck={false}
+                    />
+                    <TextField
+                        inputProps={{ 'aria-label': `Enter ${eventTitle} again` }}
+                        label='Confirm your event title to DELETE your event'
+                        helperText={errors.confirmTitle}
+                        error={Boolean(errors.confirmTitle)}
+                        required
+                        variant='outlined'
+                        value={values.confirmTitle}
+                        onChange={handleChange('confirmTitle')}
+                        spellCheck={false}
+                    />
+                </FormContent>
+                <Grid item xs={12}>
+                    <Button type='submit' variant='outlined' style={{ color: 'red', borderColor: 'red' }}>
```

Comment:
We should probably make this part of the theme.  Look at https://mui.com/customization/theming/ and maybe add a color to the palette named something like `danger` then we can apply the styles using theme.